gitlab-pipeline/stage/init.yml

```diff
@@ -105,40 +116,23 @@ init:workspace:
     -   git fetch --tags origin
     - )
 
+    # Add Mender repositories.
+    - for repo in $(integration/extra/release_tool.py --list); do
+    -   checkout_repo \
+          git@github.com:mendersoftware/$repo \
+          go/src/github.com/mendersoftware/$repo \
+          $(eval echo \$$(repo_to_env_rev $repo)) ||
```

Comment:
Question:
Isn't this exactly the same as:
```$(repo_to_env_rev $repo)``` ?

Reply:
:thinking: `$(repo_to_env_rev $repo)` will "execute" `MENDER_REV` in the subshell (for example), which will fail:

```
$ MENDER_REV=master
$ $(repo_to_env_rev mender)
MENDER_REV: command not found
```

Reply:
But only when not passed to another command, right? :)

Reply:
Yes, right. I think I rephrased that wrong.

It will result in `MENDER_REV` (in my example executing it, in the real code passing it), but what I need to result is in the value of `$MENDER_REV`, right?

```
$ function my_stuff() {
echo uno $1
echo dos $2
}
$ my_stuff something $(repo_to_env_rev mender)
uno something
dos MENDER_REV
$ my_stuff something $(eval echo \$$(repo_to_env_rev mender))
uno something
dos master
```

Reply:
Oh, right..
Then I'm with you ;)
Maybe this should just be returned from the `repo_to_env_rev` function, while we're on it?

Reply:
Okey, nice idea. Reworking...